Core/Core/Api/Operations/Operations.Serialize.cs

```diff
@@ -70,49 +83,25 @@ public static partial class Operations
       return ret ?? throw new SpeckleException($"{nameof(value)} failed to deserialize to a {nameof(Base)} object");
     }
 
-    var deserializer = new BaseObjectDeserializerV2 { CancellationToken = cancellationToken };
-    return deserializer.Deserialize(value);
+    return Deserialize(value, cancellationToken);
   }
 
-  #region obsolete
-  /// <summary>
-  /// Deserializes a list of objects into an array. Note: if you want to pull an object from speckle (either local or remote), please use any of the "Receive" methods.
-  /// </summary>
-  /// <param name="objectArr"></param>
-  /// <returns></returns>
-  [Obsolete("Please use the Deserialize(string value) function. This function will be removed in later versions.")]
+  [Obsolete("Please use the Deserialize(string value) function.", true)]
   public static List<Base> DeserializeArray(
     string objectArr,
     SerializerVersion serializerVersion = SerializerVersion.V2
   )
   {
-    if (serializerVersion == SerializerVersion.V1)
-    {
-      var (_, settings) = GetSerializerInstance();
-      return JsonConvert.DeserializeObject<List<Base>>(objectArr, settings);
-    }
-
-    var deserializer = new BaseObjectDeserializerV2();
-    List<object> deserialized = deserializer.DeserializeTransportObject(objectArr) as List<object>;
-    List<Base> ret = new();
-    foreach (object obj in deserialized)
-    {
-      ret.Add((Base)obj);
-    }
-
-    return ret;
+    throw new NotImplementedException();
```

Comment:
Several of these super old serializer functions were generating warnings. We agreed that it is acceptable to supress warnings for obsolete code. But since these functions are so old, I thought it best to remove them.

This function, was marked `obsolete` by [this commit](https://github.com/specklesystems/speckle-sharp/commit/66bad795e1a7043193e2d02610f9ade6fe1afd31) in October 2020 before SerializerV1 was ever used.

If you'd prefer I remove completely, I'd be happy to do this also, since devs have had over 3 years notice for this one 😅

Reply:
Ohhh yes I remember now. No complaints 🤐